physics-manager.js

```diff
@@ -253,6 +253,8 @@ physicsManager.simulatePhysics = timeDiff => {
   }
 };
 
+physicsManager.marchingCubes = (dims, potential, shift, scale) => physx.physxWorker.marchingCubes(dims, potential, shift, scale);

```

Comment:
I think this needs a new namespace. It may be in the same binary but it is logically different from `physics`.

Should we make a new `geometry-utils.js` for this?

Reply:
I'm wondering if I'm approved to make a new namespace. I think it should come from the design side of the project

Reply:
Yes we can make a new namespace. I'll do it...